Source/ProceduralFairings/FairingBase.cs

```diff
@@ -1376,6 +1394,8 @@ public void recalcShape ()
                         float off = sn.position.y;
                         off -= oppNodePos.y;
                         sf2.meshPos = new Vector3(-norm.magnitude, -off, 0);
+                        var currentOffset = sf2.meshPos + (openFairing ? editorOpenOffset : Vector3.zero);
+                        StartCoroutine(sf2.SetOffset(currentOffset, openSpeed));
```

Comment:
Use the nice UpdateOpen() method you made?  Or get rid of it if nothing will call it?